src/Tools/SarifValidator/SarifValidator.csproj

```diff
@@ -0,0 +1,77 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <ProjectGuid>{63384A51-476C-46B5-AF51-3BC55ACD8BB3}</ProjectGuid>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>Microsoft.CodeAnalysis.Sarif.SarifValidator</RootNamespace>
+    <AssemblyName>SarifValidator</AssemblyName>
+    <TargetFrameworkProfile />
+  </PropertyGroup>
+  <PropertyGroup>
+    <SignAssembly>true</SignAssembly>
+  </PropertyGroup>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., build.props))\build.props" />
+  <PropertyGroup>
+    <!-- Override the TargetFrameworkVersion specified in build.props. This project must target
+         .NET 4.6 because the assemblies from the SarifValidationLibraries NuGet package,
+         upon which this project depends, target that version. -->
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
+    <AssemblyOriginatorKeyFile>..\..\..\GeneratedKey.snk</AssemblyOriginatorKeyFile>
```

Comment:
Why are we signing this tool? Will we ship it in the nuget package at some point?